pkg/server/api/agent/v1/service_test.go

```diff
@@ -19,101 +21,78 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-var agent1 = "spiffe://example.org/agent-1"
-var agent2 = "spiffe://example.org/agent-2"
-var agent3 = "spiffe://example.org/agent-3"
+const (
+	agent1 = "spiffe://example.org/agent-1"
+	agent2 = "spiffe://example.org/agent-2"
+)
 
-var testNodes map[string]*common.AttestedNode = map[string]*common.AttestedNode{
-	agent1: {
-		SpiffeId:            agent1,
-		AttestationDataType: "type-1",
-		CertSerialNumber:    "CertSerialNumber-1",
-		NewCertSerialNumber: "CertSerialNumber-1",
-		CertNotAfter:        1,
-	},
-	agent2: {
-		SpiffeId:            agent2,
-		AttestationDataType: "type-2",
-		CertSerialNumber:    "CertSerialNumber-2",
-		NewCertSerialNumber: "CertSerialNumber-2",
-		CertNotAfter:        2,
-	},
-	agent3: {
-		SpiffeId:            agent3,
-		AttestationDataType: "type-2",
-		CertNotAfter:        3,
-	},
-}
+var (
+	ctx = context.Background()
 
-var testNodeSelectors map[string]*datastore.NodeSelectors = map[string]*datastore.NodeSelectors{
-	agent1: {
-		SpiffeId: agent1,
-		Selectors: []*common.Selector{
-			{
-				Type:  "node-selector-type-1",
-				Value: "node-delector-value-1",
-			},
+	testNodes = map[string]*common.AttestedNode{
+		agent1: {
+			SpiffeId:            agent1,
+			AttestationDataType: "type-1",
+			CertSerialNumber:    "CertSerialNumber-1",
+			NewCertSerialNumber: "CertSerialNumber-1",
+			CertNotAfter:        1,
 		},
-	},
-	agent2: {
-		SpiffeId: agent2,
-		Selectors: []*common.Selector{
-			{
-				Type:  "node-selector-type-2",
-				Value: "node-delector-value-2",
-			},
+		agent2: {
+			SpiffeId:            agent2,
+			AttestationDataType: "type-2",
+			CertNotAfter:        3,
 		},
-	},
-	agent3: {
-		SpiffeId: agent3,
-		Selectors: []*common.Selector{
-			{
-				Type:  "node-selector-type-3",
-				Value: "node-delector-value-3",
+	}
+
+	testNodeSelectors = map[string]*datastore.NodeSelectors{
+		agent1: {
+			SpiffeId: agent1,
+			Selectors: []*common.Selector{
+				{
+					Type:  "node-selector-type-1",
+					Value: "node-delector-value-1",
```

Comment:
This typo is in a few spots.
```suggestion
					Value: "node-selector-value-1",
```